test/v1/connectors/DeleteCharitableGivingTaxReliefConnectorSpec.scala

```diff
@@ -44,22 +45,42 @@ class DeleteCharitableGivingTaxReliefConnectorSpec extends ConnectorSpec {
         result shouldBe expectedOutcome
       }
 
-      "given a TYS request" in new TysIfsTest with Test {
+      "given a TYS request" when {
```

Comment:
Unfortunately the tests pass whether or not the 'intent' header is sent.

Reply:
This should be addressed by latest commit.

Reply:
Would it add value to the tests if we test the failed case when the feature is on but the header is excluded (+ vice versa)?